config/options.json

```
@@ -146,155 +147,179 @@
       "no": "No"
     }
   },
-  "hittableBeaks": {
-    "name": "Beak Shield Disable",
+  "mode7Fix": {
+    "name": "Mode 7 edges fix",
     "mode": "General",
     "values": {
-      "yes": "Yes"
+      "no": "No"
     }
   },
-  "percentPoison": {
-    "name": "Percentage Damage Poison",
+  "mechRiderDeathFix": {
+    "name": "Mech rider death fix",
+    "mode": "General",
+    "values": {
+      "no": "No"
+    }
+  },
+  
+  "aggBosses": {
+    "name": "Aggressive bosses",
     "mode": "General",
     "values": {
       "yes": "Yes"
     }
   },
-  "defRefactor": {
-    "name": "Defense Refactor",
+  "aggEnemies": {
+    "name": "Aggressive enemies",
     "mode": "General",
     "values": {
       "yes": "Yes"
     }
   },
-  "moreItems": {
-    "name": "7 items max",
+  "noBsVampires": {
+    "name": "Hittable Vampires",
```

Only first letter should be uppercase and the rest should be in lowercase to remain consistent throughout options...

```suggestion
    "name": "Hittable vampires",
```